include/st_convert_api.h

```diff
@@ -269,6 +269,77 @@ static inline int st20_rfc4175_422be12_to_422le12(
   return st20_rfc4175_422be12_to_422le12_simd(pg_be, pg_le, w, h, ST_SIMD_LEVEL_MAX);
 }
 
+/**
+ * Convert rfc4175_444be10 to yuv444p10le with the max optimised SIMD level.
+ *
+ * @param pg
+ *   Point to pg(rfc4175_444be10) data.
+ * @param y
+ *   Point to Y(yuv444p10le) vector.
+ * @param b
+ *   Point to b(yuv444p10le) vector.
+ * @param r
+ *   Point to r(yuv444p10le) vector.
+ * @param w
+ *   The st2110-20(video) width.
+ * @param h
+ *   The st2110-20(video) height.
+ * @return
+ *   - 0 if successful.
+ *   - <0: Error code if convert fail.
+ */
+static inline int st20_rfc4175_444be10_to_yuv444p10le(
+    struct st20_rfc4175_444_10_pg4_be* pg, uint16_t* y, uint16_t* b, uint16_t* r,
+    uint32_t w, uint32_t h) {
+  return st20_rfc4175_444be10_to_444p10le_simd(pg, y, b, r, w, h, ST_SIMD_LEVEL_MAX);
+}
+
+/**
+ * Convert rfc4175_444be10 to gbrp10le with the max optimised SIMD level.
+ *
+ * @param pg
+ *   Point to pg(rfc4175_444be10) data.
+ * @param g
+ *   Point to g(gbrp10le) vector.
+ * @param b
+ *   Point to b(gbrp10le) vector.
+ * @param r
+ *   Point to r(gbrp10le) vector.
+ * @param w
+ *   The st2110-20(video) width.
+ * @param h
+ *   The st2110-20(video) height.
+ * @return
+ *   - 0 if successful.
+ *   - <0: Error code if convert fail.
+ */
+static inline int st20_rfc4175_444be10_to_gbrp10le(struct st20_rfc4175_444_10_pg4_be* pg,
+                                                   uint16_t* g, uint16_t* b, uint16_t* r,
+                                                   uint32_t w, uint32_t h) {
+  return st20_rfc4175_444be10_to_444p10le_simd(pg, g, b, r, w, h, ST_SIMD_LEVEL_MAX);
```

Comment:
Little confused here, YUV and RGB use different color space, can we directly assign gbrp10le to 444p10le? Or you mean the input format is the rgb10be in the rfc4174 spec which use same layout to 444be10?